src/components/Search.jsx

```diff
@@ -99,7 +99,6 @@ export const Search = ({
         closeMenu,
     } = useCombobox({
```

Comment:
https://www.npmjs.com/package/downshift#advanced-props

Seems, we need inputValue for the useCombobox to store the search value, and the issue that is mentioned in the ticket seems the cursor moving to the end of the text as soon as we type. I think removing of inputValue is working since we are doing API call for search results, but it won't work for static items.